### DISCUSSION TITLE
snapshot becomes ready to use but never shows up on the backup tab

### DISCUSSION THREAD

**Discussion by bradwilson331** [longhorn/longhorn#4958] [Q&A]
When creating a snapshot/backup I never see them move to the backup tab so they can be used for a restore or in the nfs share. I do not have traffik2 installed, only metal-lb. Any troubleshooting procedures would be welcome.
[longhorn-support-bundle_e0549e73-411a-46e1-bd96-eb7387c44951_2022-11-29T04-17-43Z.zip](https://github.com/longhorn/longhorn/files/10109692/longhorn-support-bundle_e0549e73-411a-46e1-bd96-eb7387c44951_2022-11-29T04-17-43Z.zip)

**Comment by derekbit** [comment]
Permission denied error found in your support bundle.
`Error creating backup: mkdir /var/lib/longhorn-backupstore-mounts/192_168_7_200/dev/backup_k8s/backupstore: permission denie`

Can you please check the permission setting of your remote storage?

**Reply by bradwilson331** [reply]
Awesome! Thanks for the help!  Still learning this platform. Would that error be visible in the Longhorn logs?

**Reply by derekbit** [reply]
Do you mean on the UI? I think no. We should improve this part later.
